source/_components/telegram_bot.markdown

```diff
@@ -11,12 +11,132 @@ logo: telegram.png
 ha_release: 0.42
 ---
 
-Use Telegram on your mobile device to send messages or commands to your Home Assistant.
+Use Telegram on your mobile or desktop device to send and receive messages or commands to/from your Home Assistant.
 
+This component creates notification services to send, or edit previously sent, messages from a [Telegram Bot account](https://core.telegram.org/bots) configured either with the [polling](/_components/telegram_bot.polling.markdown) method or with the [webhooks](/_components/telegram_bot.webhooks.markdown) one, and trigger events when receiving messages.
```

Comment:
The links for polling and webhooks are incorrect. They should target:
`/components/telegram_bot.polling/`
`/components/telegram_bot.webhooks/`